src/internal/connector/support/m365Transform_test.go

```diff
@@ -37,7 +37,7 @@ func (suite *SupportTestSuite) TestToMessage() {
 	assert.NotEqual(t, message.GetId(), clone.GetId())
 }
 
-func (suite *SupportTestSuite) TestToEventSimplified() {
+func (suite *SupportTestSuite) TestToEventSimplifiedAttendees() {
```

Comment:
Fyi, the Golang test naming convention we want to follow is `Test<FuncOrStructName>_<MethodNameIfExists>_<conditionIfExists>()`.

So in this case, a more appropriate name would be:
```suggestion
func (suite *SupportTestSuite) TestToEventSimplified_attendees() {
```